server/app/services/startup_service.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timedelta
+from tabnanny import check
+from app.models.check import Check
+from app.controllers.checks_controller import ChecksController
+
+from app import scheduler
+
+class StartUpService:
+
+    @staticmethod
+    def schedule_jobs():
```

Comment:
In general, I would prefer a method that loops over the set of jobs, scheduling each one.  That is more o-o.